gbpservice/nfp/scripts/mode_shift_enterprise.sh

```diff
@@ -2,27 +2,43 @@
 
```

Comment:
Lets rename the file to mode_shift.sh as in future, this can be used for other mode shifts also.